company-clang.el

```diff
@@ -80,6 +80,76 @@ or automatically through a custom `company-clang-prefix-guesser'."
     (insert-file-contents-literally file nil beg end)
     (buffer-string)))
 
+(defcustom company-clang-parse-documentation nil
+  "When non-nil, clang will parse completion's comments/documentation."
+  :type 'boolean)
+
+(defcustom company-clang-documentation-fill-column 70
+  "Column beyond which automatic line-wrapping should happen."
+  :type 'integer)
+
+(defcustom company-clang-documentation-justify 'full
+  "Specifies which  kind of justification to do."
+  :type '(choice (const :tag "Full" full)
+		 (const :tag "Left" left)
+		 (const :tag "Right" right)
+		 (const :tag "Center" center)
+                 (const :tag "None" none)))
+
+(defvar company-clang-summary-list
```

Comment:
I'd say this should include the default value (nil), but this variable probably shouldn't be used at all, see below.